.github/workflows/publish-npm-rc.yaml

```diff
@@ -0,0 +1,62 @@
+name: Publish release candidate
+
+on:
+  push:
+    branches: ['releases/[0-9]+.[0-9]+.[0-9]+-rc[0-9]+']
```

Comment:
Perhaps we could just do `release/*` (also should probably be release, not releases)

See other comments below